resources/references/app-reference/flow-action-reference.md

```diff
@@ -0,0 +1,51 @@
+# Flow Action Reference
+
+{% code title="flow-action.xml" %}
+```xml
+<flow-actions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/FlowAction/Schema/flow-action-1.0.xsd">
+    <flow-action>
+        <meta>
+            <name>slack.send.message</name>
+            <badge>Slack</badge>
+            <label>Send slack message</label>
+            <label lang="de-DE">Send slack message</label>
+            <description>Slack send message description</description>
+            <description lang="de-DE">Slack send message description DE</description>
+            <url>https://hooks.slack.com/services/JHUWIHMSDSKAJBU868/JDUIGDSABDKJKAFT87687</url>
```

Comment:
i guess we should no leak those ids? better use a {id} placeholder as you did in the guide